contracts/v1.5.x/CoreWallet.sol

```diff
@@ -673,6 +675,19 @@ contract CoreWallet is IERC1271 {
         internalInvoke(operationHash, _data);
     }
 
+    /// @notice simulate internalInvoke result off-chain
+    /// @dev this function will revert always
+    /// @param _nonce the nonce value for the signature
+    /// @param _data The data containing the transactions to be invoked; see internalInvoke for details.
+    function simulateInvoke(uint256 _nonce, bytes calldata _data) external {
+        // check nonce
+        require(_nonce > nonce && (_nonce < (nonce + 10)), "must use valid nonce");
```

Comment:
guess no need, but need to compensate gas consumption

Reply:
add sig validation operation for gas estimation

Reply:
update to simulateInvoke2, let it same as invoke2 but remove checking